Decred Wallet/Custom Views/UIPaddedLabel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIPaddedLabel.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+
+import UIKit
+
+public class UIPaddedLabel: UILabel {
```

Comment:
`PaddedLabel` resonates more with other custom views we have, we've not been using the `UI-` prefix.